src/react/projects/spark-core-react/src/SprkOrderedList/SprkOrderedList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+const SprkOrderedList = (props) => {
+  const { children, listType, additionalClasses, idString, ...other  } = props;
+  const classNames = classnames(
+    'sprk-b-List',
+    additionalClasses, {
+      'sprk-b-List--indented': listType === 'indented'
+    },
+  )
+
+  return (
+    <ol className={classNames} data-id={idString} {...other}>
```

Comment:
is the only difference the ol / ul ? If so, we could combine them doing something like:
``` 
       const TagName = props.isOrdered ? 'ol' : 'ul';
       render(<TagName></TagName>);

Reply:
I followed the way the Angular component is built. The only other difference is that a ul can be bare or indented and an ol can be indented but not bare. I can certainly combine the two if we want to do it that way.